containers/osbuild-composer/entrypoint.py

```diff
@@ -286,14 +305,20 @@ def run(self):
             if self.args.dnf_json:
                 proc_dnf_json = self._spawn_dnf_json()
 
-            proc_composer = self._spawn_composer(sockets)
+            if any([self.args.weldr_api, self.args.composer_api, self.args.local_worker_api, self.args.remote_worker_api]):
```

Comment:
These modifications are made to allow dnf_json to be started without composer ?

Reply:
Yea exactly. We'd only wanna run dnf-json in some cases.